### ISSUE TITLE
Still using?

### ISSUE THREAD

**Issue by km-pg** [rdegges/django-twilio#97]
Poll: who's still using this project in your apps? Respond here.

**Comment by mathieusteele** [comment]
I have it in testing and it is not used in production. I recently upgraded
Django and am experiencing some issues with compatibility.

**Comment by yousifa** [comment]
Want to use, but compatibility issues w/ 1.11, trying to fix

**Comment by jheld** [comment]
Yep using for our project with django 1.11, currently haven't noticed any issues, but we're also on v5. Need v6 support really soon.

**Comment by wbt** [comment]
http://govtrack.us/ is still using this, assuming this is the code referred to on line 20 of [this file](https://github.com/govtrack/govtrack.us-web/blob/master/build/pipreq.txt), installed with pip.

**Comment by birdsarah** [comment]
@tomd-aptivate - is ilsss still active / using this?

**Comment by marauder37** [comment]
We're using this in production, just ran into compatibility problems

**Comment by andrewheyer** [comment]
Using it in production, was getting deprecation issues. Adding app labels to the meta class seemed to have fixed it.

**Comment by powerhouseofthecell** [comment]
Still using it in personal apps, running into problems with pip-installed decorators.py file

**Comment by jheld** [comment]
We're using the newer version of this. Haven't noticed any glaring issues (silence can be a good thing). Texts and phone calls seem to work great.

**Comment by mulka** [comment]
I've pushed a forked version of django-twilio to pypi as django-twilio2. Please take a look and let me know of any issues you run into.

**Comment by rdegges** [comment]
Hi guys,

I've said this a few times, but I'd be happy to pass over ownership of this project to someone else. (@mulka -- if you're maintaing a fork would you like direct access here so you can manage the official one?)

My situation is that I haven't used Django full-time for a while now, and don't have any active django-twilio projects. Because of this (and other time commitments) I haven't maintained this in a while. Paul from Twilio took over maintenance a while ago, but when he left Twilio he stopped working on it as well.

Please hit me up, as I'd be moreeee than happy to pass over maintenance, admin creds, PyPI, etc. to someone who wants to run it!

**Comment by jheld** [comment]
Hi all,

I'm a new contributor/maintainer of this project. Thanks @rdegges. I'll be looking through the issues and PRs as quickly as I can. Please provide any updates (usage, issues, workarounds) that you may have, or patches, as PRs; otherwise it's in order of importance, followed by morality.

**Comment by kblicharski** [comment]
@jheld Myself and a coworker are currently working through a mountain of tech debt, and in the process, want to upgrade to Django 2.0 and Twilio 6, but are being limited by this project. Let me know if there's a way we can help with some of the issues here, like #101 or #103.

**Comment by jheld** [comment]
Hi @kblicharski,

I did merge #101 into develop, and a couple other PRs. But yes there is indeed more work to do.

I'll take a look at #103. Hopefully it's as simple as the issue suggests.

I'm waiting on PyPI credentials so general access to the next release will be available.

I'll probably update master soon though and make a version bump.

I'll let you know if I need any help. Thanks!

**Comment by jheld** [comment]
Please check out the latest alpha build on PyPI or build it from master.

**Comment by jheld** [comment]
Hey everyone.

I've released version `0.9.0` to [PyPI](https://pypi.python.org/pypi?:action=display&name=django-twilio&version=0.9.0). It is nearly identical to `0.9.0a1` (docs, some test case and module import improvements). My company has been using it for a week and so far we've encountered no issues. I'll update via an issue and MR if we do find anything.

Because this project is not yet `1.0`, and because the fixes going into `0.9.0` have been tested in various forms over the last half year (and been in use on `django-twilio2`), I decided not to have beta or release candidate releases. I know you and other API users have been wanting this project to continue to work well.

In the future, I anticipate that we'll design a workflow and cadence for releases. Once we hit `1.0.0`, that will be a high priority.

Please upgrade :) As usual, please open any issues as you see them, and if able, any MRs.

As such, I'm closing this issue. We can re-open it, but if possible, it would be nice to have a new one with any specific issues that are encountered (if any).
